### PR TITLE
705 Add slack-notifier gem

### PR DESCRIPTION
**Pull request by SamJamCul**
### Context
https://trello.com/c/2Fc8tzQ0/705-s-m-configure-for-slack-notifications

### CODE REVIEW

**Comment by SamJamCul** [comment]
[[S/M] Configure for slack notifications](https://trello.com/c/2Fc8tzQ0/705-s-m-configure-for-slack-notifications)

**Review by dankmitchell — APPROVED** [review]

File: app/services/slack_notifier_service.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "slack-notifier"
+
+class SlackNotifierService
+  include ServicePattern
+
+  def initialize(channel: "#twd_publish_register_tech", message:)
```
Comment (misaka) [inline app/services/slack_notifier_service.rb:8]: Would it make sense to put this into `Settings`?

File: app/services/slack_notifier_service.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    @channel = channel
+    @message = message
+    @notifier = Slack::Notifier.new(ENV.fetch("SLACK_WEBHOOK_URL"))
```
Comment (misaka) [inline app/services/slack_notifier_service.rb:11]: Traditionally we use `Settings` for this kind of configuration, gives us a unified approach to that. Is there a reason `SLACK_WEBHOOK_URL` shouldn't be in there?

Comment (SamJamCul) [reply app/services/slack_notifier_service.rb:11]: Nope, will change!

File: app/services/slack_notifier_service.rb (hunk 1 above)
Comment (SamJamCul) [reply app/services/slack_notifier_service.rb:8]: Sure thing!

**Pull request merged**